apps/etterlatte-behandling/src/main/kotlin/config/BeanFactory.kt

```diff
@@ -293,4 +301,17 @@ class EnvBasedBeanFactory(private val env: Map<String, String>) : CommonFactory(
 
         return Norg2KlientImpl(client, env.getValue("NORG2_URL"))
     }
+
+    override fun featureToggleService(): FeatureToggleService {
+        return FeatureToggleService.initialiser(
+            mapOf(
+                FeatureToggleServiceProperties.ENABLED.navn to config.getString("funksjonsbrytere.enabled"),
```

Comment:
Kanskje vi også skal la enabled defaulte til true, tanken med den er kun for å opne for testing.

Reply:
Lurer på om vi burde bruke FakeUnleash til testing - men - håper vi kan ta dette som en separat endring